OKTableViewLiaison/Classes/Enums/OKTableViewSectionComponentDisplayOption.swift

```diff
@@ -15,10 +15,6 @@ public enum OKTableViewSectionComponentDisplayOption {
     case both(headerComponent: OKAnyTableViewSectionComponent,
```

Comment:
Even on the smallest screen, `case both(headerComponent: OKAnyTableViewSectionComponent, footerComponent: OKAnyTableViewSectionComponent)` reads better than the line break

Reply:
So your saying remove the line break for both?